packages/airnode-deployer/src/infrastructure/index.test.ts

```diff
@@ -1521,3 +1534,288 @@ describe('deploymentInfo', () => {
     expect(gcpGetFileFromBucketSpy).not.toHaveBeenCalled();
   });
 });
+
+describe('fetchFiles', () => {
+  const bucket = {
+    name: 'airnode-123456789',
+    region: 'us-east-1',
+  };
+  const configPath = path.join(__dirname, '..', '..', 'test', 'fixtures', 'config.aws.valid.json');
```

Comment:
Nit: You can just do `path.join(__dirname, '../../test/fixtures/config.aws.valid.json')`